app/src/main/res/layout/item_media_preview.xml

```diff
@@ -110,6 +110,25 @@
             app:srcCompat="@drawable/ic_eye_24dp"
             app:tint="@color/white" />
 
+        <TextView
```

Comment:
I see that other elements here seem to be duplicated in the report view.

I'm not sure if that is desired also for this element?

Reply:
I don't think it's that important in the report